src/main/java/com/amazon/jenkins/ec2fleet/EC2FleetCloud.java

```diff
@@ -163,6 +164,27 @@ public class EC2FleetCloud extends AbstractEC2FleetCloud {
     private transient AtomicInteger plannedNodeCounter = new AtomicInteger(1);
 
     @DataBoundConstructor
+    public EC2FleetCloud(@Nonnull final String name,
+                         final @Deprecated String credentialsId,
```

Comment:
Does `credentialsId` need to be part of this constructor? 🤔

Reply:
Not sure, I left it there for backwards compatibility with our tests and because that was was how I interpreted it in [CasC support guidelines](https://github.com/jenkinsci/configuration-as-code-plugin/blob/master/docs/PLUGINS.md).